autoload/denops/_internal/job.vim

```diff
@@ -45,7 +45,7 @@ if has('nvim')
     call a:callback(a:job, a:status, a:event)
   endfunction
 else
-  " https://github.com/neovim/neovim/blob/f629f83/src/nvim/event/process.c#L24-L26
+  " https://github.com/neovim/neovim/blob/cb24a3907c8d24a898d99042f0f16c8919a2e7ab/src/nvim/event/process.c#L28
```

Comment:
It's OK but why do you change the URL? The commit message said it does not exist but it actually exist.

https://github.com/neovim/neovim/blob/f629f83/src/nvim/event/process.c#L24-L26

Reply:
Strangely, it only seems to 404 when viewed from my GitHub account (even after clearing the cache).

Reply:
I'll put it back for now.